src/cookies.ts

```diff
@@ -0,0 +1,29 @@
+import { Response, Request } from 'express';
+
+export { setCookie, getCookie };
+
+function setCookie({
+  res,
+  name,
+  value,
+}: {
+  res: Response;
+  name: string;
+  value: any;
+}) {
+  res.cookie(name, JSON.stringify(value), {
+    encode: String,
+  });
+}
+
+function getCookie({
+  req,
+  name,
+  defaultValue,
+}: {
+  req: Request;
+  name: string;
+  defaultValue: any;
+}) {
+  return req.cookies[name] ? JSON.parse(req.cookies[name]) : defaultValue;
```

Comment:
`JSON.parse` can error. Consider using try catch and potentially passing in `res` so that the cookie can be cleared, because the cookie is malformed at this point.